front/src/routes/dashboard/style.css

```
@@ -361,6 +361,39 @@
+   activity feed errors) sit directly on the scene: same tinted notice family,
+   on an opaque pastel so they stay readable over the glows. The in-card rules
+   above are more specific and keep their tighter metrics. */
+:global(.glass-theme .alert) {
+  border: none;
+  border-radius: var(--gl-radius-md);
+  font-weight: 500;
+  background-color: rgba(255, 255, 255, 0.55);
+  color: #667081;
+}
+
+:global(.glass-theme .alert-danger) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)